toolbox

```diff
@@ -347,6 +347,17 @@ create_environment_options()
 }
 
 
+set_terminal_size()
```

Comment:
Maybe move this into `create_environment_options` itself?

Reply:
Sure. it could be.